### ISSUE TITLE
508-defect-2: Address of outpatient behavioral clinic is missing on make an appointment

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#21361]
Label: 20132

    - [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

Issue Title: Make an appointment, address of outpatient behavioral clinic missing
 Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela 

<h2>User Story or Problem Statement:</h2>
As a person with a disability who depends on public transportation, it is essential that I have the exact address of my destination when booking rideshare or para transit, or planning a trip using the city bus.

<h2>Details</h2>
On the "Make an appointment" page, under the level 3 heading "Walk in Mental Health Care," the address of the Salt Lake City VA Medical Center outpatient behavioral clinic is missing. It is available on the map that is linked, but the link text does not indicate this.
Acceptance Criteria

Environment

Steps to Recreate

<h2>Proposed Solution (if known)</h2>
Ideally, include the address in the text under the level 3 heading "For walk in mental health care."
Alternatively, indicate in the link text that the address is located on the page with the map.

WCAG or Vendor Guidance (optional)

<h2>Screenshots or Trace Logs</h2>

![image](https://user-images.githubusercontent.com/78870095/110689299-65080000-8197-11eb-9d48-fd1a972b7601.png)
